README.md

```diff
@@ -1,32 +1,127 @@
-# Gradle Lifecycle
+# Lifecycles, phases, and goals
```

Comment:
Shouldn't this be Lifecycles, phases, and tasks instead of goals?

Also, should we specify "Gradle" in here and "Maven" in the other (similar) lesson to make it easier to differentiate in Canvas?

Reply:
I updated both maven and gradle lessons to have unique titles, also updated the learning goals to be consistent between both.